ceph.spec.in

```diff
@@ -650,6 +650,23 @@ ceph-osd is the object storage daemon for the Ceph distributed file
 system.  It is responsible for storing objects on a local file system
 and providing access to them over the network.
 
+%if 0%{with seastar}
+%package crimson-osd
+Summary:	Ceph Object Storage Daemon
```

Comment:
This would mean, we would have two packages with the same `Summary`, right?

Reply:
yeah. i think i should have put more background here:

the long term plan is to rename to crimson-osd to ceph-osd. and let it take the place of ceph-osd. before that happens, this packaging will only be consumed by teuthology for testing crimson-osd.

Reply:
@b-ranto i revised the summary to avoid the confusion in future.